<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Fortune Br - Bill of Materials</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/favicon.ico</normaloff>icons/favicon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{ background-color:rgb(202, 234, 255) }

QMenuBar::item:selected{ background: rgb(235, 248, 255) }

QMenu::item:selected {  background:rgb(65, 163, 255) }

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QPushButton::pressed {  background:rgb(144, 198, 255) }
</string>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>341</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(255, 255, 255)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableView" name="tableView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>2</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(255, 255, 255)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>308</width>
      <height>82</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>60</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Fixedsys</family>
         <pointsize>13</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Show</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>60</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Fixedsys</family>
         <pointsize>13</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Export</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>130</y>
      <width>301</width>
      <height>351</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_sth">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(217, 255, 215)</string>
       </property>
       <property name="text">
        <string>Stitching Charge</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_Vstitch">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(217, 255, 215)</string>
       </property>
       <property name="text">
        <string>0.00</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_print">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(217, 255, 215)</string>
       </property>
       <property name="text">
        <string>Printing Charge</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_Vprint">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(217, 255, 215)</string>
       </property>
       <property name="text">
        <string>0.00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_mrp">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(229, 255, 190)</string>
       </property>
       <property name="text">
        <string>MRP</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_Vmrp">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(229, 255, 190)</string>
       </property>
       <property name="text">
        <string>0.00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_basic">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(229, 255, 190)</string>
       </property>
       <property name="text">
        <string>BASIC</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_Vbasic">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(229, 255, 190)</string>
       </property>
       <property name="text">
        <string>0.00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_cop">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(227, 255, 185)</string>
       </property>
       <property name="text">
        <string>Cost of Production</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_Vcop">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(227, 255, 185)</string>
       </property>
       <property name="text">
        <string>0.00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_netm">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(240, 255, 160)</string>
       </property>
       <property name="text">
        <string>Net Margin</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="label_Vnetm">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(240, 255, 160)</string>
       </property>
       <property name="text">
        <string>0.00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuDatabase">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Database</string>
    </property>
    <widget class="QMenu" name="menuBom">
     <property name="title">
      <string>Bom</string>
     </property>
     <addaction name="actionUpdate_From_File"/>
    </widget>
    <widget class="QMenu" name="menuOS_Charges">
     <property name="title">
      <string>OS Charges</string>
     </property>
     <addaction name="actionAdd_New_Charges"/>
     <addaction name="actionUpdate_Existing_Charges"/>
     <addaction name="separator"/>
     <addaction name="actionUpdate_From_File_2"/>
    </widget>
    <widget class="QMenu" name="menuPrice_Structure">
     <property name="title">
      <string>Price Structure</string>
     </property>
     <addaction name="actionAdd_New_Price"/>
     <addaction name="actionUpdate_Existing_Price"/>
     <addaction name="separator"/>
     <addaction name="actionUpdate_From_File_3"/>
    </widget>
    <addaction name="menuBom"/>
    <addaction name="menuOS_Charges"/>
    <addaction name="menuPrice_Structure"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDatabase"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionUpdate_From_File">
   <property name="text">
    <string>Update From File</string>
   </property>
  </action>
  <action name="actionAdd_New_Charges">
   <property name="text">
    <string>Add New Charges</string>
   </property>
  </action>
  <action name="actionUpdate_Existing_Charges">
   <property name="text">
    <string>Update Existing Charges</string>
   </property>
  </action>
  <action name="actionUpdate_From_File_2">
   <property name="text">
    <string>Update From File</string>
   </property>
  </action>
  <action name="actionAdd_New_Price">
   <property name="text">
    <string>Add New Price</string>
   </property>
  </action>
  <action name="actionUpdate_Existing_Price">
   <property name="text">
    <string>Update Existing Price</string>
   </property>
  </action>
  <action name="actionUpdate_From_File_3">
   <property name="text">
    <string>Update From File</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
